docs/userGuide/components/pic.md

```diff
@@ -24,6 +24,6 @@
 Name | Type | Default | Description 
 --- | --- | --- | ---
 alt | `string` | | The alternative text of the image.
-height | `string` | |
+height | `string` | | The height of the image, using a [CSS units](https://www.w3schools.com/cssref/css_units.asp).
 src | `string` | |
-width | `string` | |
+width | `string` | | The width of the image, using a [CSS units](https://www.w3schools.com/cssref/css_units.asp).

```

Comment:
Default HTML5 standard has `width` and `height` specified as just numbers then will be rendered using `px` unit. Need to mention `alt` and `src` must be specified, as it is also a HTML5 standard.